integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -487,6 +487,41 @@ public void testHourOfDayWithUnderscores() throws IOException {
     verifyDataRows(result, rows(17));
   }
 
+  @Test
+  public void testExtractWithDatetime() throws IOException {
+    JSONObject datetimeResult = executeQuery(
+        String.format("SELECT extract(DAY_SECOND FROM datetime0) FROM %s LIMIT 1", TEST_INDEX_CALCS));
```

Comment:
Consider adding cast to these tests, because all datetime columns are `TIMESTAMP`.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/221/commits/7f018333ef2d1002180ecdd9638e17c4e99ab096